src/dummyEngine.ts

```diff
@@ -0,0 +1,31 @@
+// buttons available in beginner mode
+export const beginnerModeButtons: string[] = [
+    "Help",
+    "Play",
+    "Play Slowly",
+    "Play Step by Step",
+    "Stop",
+    "New Project",
+    "Save",
+    "Merge",
+    "Load",
+    "Restore",
+    "Language Selection"
+];
+
+// buttons available in advanced mode
+export const advancedModeButtons: string[] = [
+    "Enable Horizontal Scroll"
+]
```

Comment:
we don't want to have modes. Some buttons will be easily accessible and a few a little out of sight.